zeebe/logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/log/LogStreamBuilderImpl.java

```diff
@@ -24,6 +26,7 @@ public final class LogStreamBuilderImpl implements LogStreamBuilder {
   private InstantSource clock;
   private Limit requestLimit;
   private RateLimit writeRateLimit;
+  private MeterRegistry meterRegistry = new SimpleMeterRegistry();
```

Comment:
:question: is this really needed? This may cause hard to diagnose bugs down the line if the method `withMeterRegistry` is not called (which is a real possibility). If it's a required dependency we should add it to the builder constructor.
I see that it's the same as in `stable/8.5`, but it's probably worth changing it if does not implies too many changes

Reply:
I think in 8.5 it's because there were tons of calls to the constructor and so on. It's much more limited on main, so let's go for it.